packages/flame/lib/src/components/component_set.dart

```diff
@@ -36,9 +36,10 @@ class ComponentSet extends QueryableOrderedSet<Component> {
   final Component parent;
 
   ComponentSet(
-    int Function(Component e1, Component e2)? compare,
-    this.parent,
-  ) : super(compare);
+    int Function(Component e1, Component e2)? comparator,
+    this.parent, {
+    bool strictMode = true,
```

Comment:
Why is the default true here, and false in `createDefault`?

Reply:
my bad, should be true everywhere